packages/client/README.md

```diff
@@ -0,0 +1,69 @@
+# `@lens-protocol/client`
+
+This package contains low level client for GraphQL Lens API.
+
+**Its interface will change without notice, use it at your own risk.**
+
+## Examples
+
+For all exaples below we require the same setup, we need a LensClient instance and an ethers.js Wallet as a signer.
+
+```ts
+import LensClient, { mumbai } from '@lens-protocol/client';
+import { Wallet } from 'ethers';
+
+const lensClient = LensClient.init(mumbai);
```

Comment:
as there is only one way to init the client, this could also just be

```
const lensClient = new LensClient(mumbai);
```

Reply:
@cesarenaldi @reecejohnson @desfero any opinions on the constructor?

Reply:
no strong opinion on constructor vs. static factory function now that we only have one. I agree with @desfero to make the argument and object could be beneficial if we are to add more non-env related config.